node.c

```diff
@@ -0,0 +1,74 @@
+#include "node.h"
+#include "board.h"
+
+
+void initializeRoot(node* root, struct board *boardState){
+
+    // Setting default values
+    root->parent = NULL;
+    root->childMoves = (int *)malloc(2 * sizeof(int));
+    root->numChildren = 0;
+    root->numWins = 0;
+    root->board = *boardState;
```

Comment:
You should use copyBoard() to copy the passed board into your node.

 - [ ] Use copyBoard() in initializeRoot